tests/test_examples.py

```diff
@@ -185,6 +185,12 @@ def test_svm_batch(self):
      'decision_forest_regression_batch.csv', lambda r: r[1].prediction, (2023, 'P', 1)),
     ('decision_forest_regression_hist_batch',
      'decision_forest_regression_batch.csv', lambda r: r[1].prediction, (2023, 'P', 1)),
+    ('decision_forest_regression_default_dense_batch',
+     'decision_forest_regression_batch_20230101.csv',
+     lambda r: r[1].prediction, (2023, 'P', 101)),
+    ('decision_forest_regression_hist_batch',
+     'decision_forest_regression_batch_20230101.csv',
+     lambda r: r[1].prediction, (2023, 'P', 101)),
```

Comment:
This set of examples will fail when version updates from 2023.1.0 to 2023.1.1 because two sets of regression examples will be run.